soap/models.py

```
@@ -0,0 +1,49 @@
+    metadata = models.JSONField(default=dict)
+    generated_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
+
+    class Meta:
+        verbose_name = 'گزارش SOAP'
+        verbose_name_plural = 'گزارش‌های SOAP'
+        indexes = [
+            models.Index(fields=['encounter_id']),
+            models.Index(fields=['patient']),
+            models.Index(fields=['doctor']),
+        ]
```

Ruff (0.12.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)